src/psyclone/tests/psyclone_test_utils.py

```diff
@@ -289,10 +335,11 @@ def get_invoke(algfile, api, idx=None, name=None):
         dir_name = "dynamo0p3"
     elif api == "gocean0.1":
         dir_name = "gocean0p1"
-    elif api == "dyanmo0.1":
+    elif api == "dynamo0.1":
         dir_name = "dynamo0p1"
     else:
-        assert False
+        raise RuntimeError("The API '{0}' is not supported by get_invoke.".
```

Comment:
Thanks for improving this bit. Please could you extend the error message to tell the user which APIs are supported.